Makefile

```diff
@@ -205,22 +210,26 @@ deploy-inventory-service-file: deploy-namespace
 		--profile "$(PROFILE)" --apply-manifest $(APPLY_MANIFEST)
 	sleep 5;  # wait for service to get an address
 
-deploy-service-requirements: deploy-namespace deploy-inventory-service-file
+deploy-service-requirements: | deploy-namespace deploy-inventory-service-file
 	python3 ./tools/deploy_assisted_installer_configmap.py --target "$(TARGET)" --domain "$(INGRESS_DOMAIN)" \
 		--base-dns-domains "$(BASE_DNS_DOMAINS)" --namespace "$(NAMESPACE)" --profile "$(PROFILE)" \
 		$(INSTALLATION_TIMEOUT_FLAG) $(DEPLOY_TAG_OPTION) --auth-type "$(AUTH_TYPE)" --with-ams-subscriptions "$(WITH_AMS_SUBSCRIPTIONS)" $(TEST_FLAGS) \
 		--ocp-versions '$(subst ",\",$(OPENSHIFT_VERSIONS))' --public-registries "$(PUBLIC_CONTAINER_REGISTRIES)" \
 		--check-cvo $(CHECK_CLUSTER_VERSION) --apply-manifest $(APPLY_MANIFEST) $(ENABLE_KUBE_API_CMD) $(E2E_TESTS_CONFIG) \
 		--ipv6-support $(IPV6_SUPPORT)
+	$(MAKE) deploy-role deploy-resources
 
 deploy-resources: generate-manifests
 	python3 ./tools/deploy_crd.py $(ENABLE_KUBE_API_CMD) --apply-manifest $(APPLY_MANIFEST) --profile "$(PROFILE)" --target "$(TARGET)"
 
-deploy-service: deploy-namespace deploy-service-requirements deploy-role deploy-resources
+deploy-service: deploy-service-requirements
 	python3 ./tools/deploy_assisted_installer.py $(DEPLOY_TAG_OPTION) --namespace "$(NAMESPACE)" \
 		--profile "$(PROFILE)" $(TEST_FLAGS) --target "$(TARGET)" --replicas-count $(REPLICAS_COUNT) \
 		--apply-manifest $(APPLY_MANIFEST) \
 		$(ENABLE_KUBE_API_CMD)
```

Comment:
You removed the parameter from the deployment of the service then you don't need it here as well.